### PR TITLE
Add NoHeader() to TableWidget

### PR DESCRIPTION
**Pull request by JetSetIlly**
Discussion here:
https://github.com/AllenDang/giu/discussions/829

The SimpleHeader() idea in the first post is too inflexible. This pull request is for the NoHeader() idea from the second post.

### CODE REVIEW

File: TableWidgets.go [hunk 1]
```diff
@@ -132,6 +134,12 @@ func (t *TableWidget) FastMode(b bool) *TableWidget {
 	return t
 }
 
+// NoHeader indicates that the column header should not be shown.
```
Comment (gucio321) [inline TableWidgets.go:137]: Add here that "it allows to setup column flags in `Columns` method" and say something about this method in a comment over Columns method - just for documentation

**Review by gucio321 — CHANGES_REQUESTED** [review]
Ifv that works for you i think it is ok. Just a small change in comments pls

File: TableWidgets.go (hunk 1 above)
Comment (JetSetIlly) [reply TableWidgets.go]: Okay. New change committed

**Review by gucio321 — APPROVED** [review]
Lgtm
@AllenDang could you take a look and merge

**Pull request merged**